app/Http/Controllers/API/ConversationsController.php

```
@@ -26,11 +28,32 @@ public function __construct()
     /**
      * Display a listing of the resource.
      *
-     * @return \Illuminate\Http\Response
+     * @return ConversationCollection
      */
     public function index()
     {
-        $conversations = Conversation::paginate(50);
+        $conversations = Conversation::where('status', '!=', ConversationNode::ARCHIVED)->paginate(50);
+
+        foreach ($conversations as $conversation) {
+            $conversation->outgoing_intents = $this->outgoingIntents($conversation);
+            $conversation->opening_intent = $this->openingIntent($conversation);
+
+            $conversation->makeVisible('id');
```

Can we do the work to make id visible by default and hide it for the conversation extraction command

---

RE: https://github.com/opendialogai/opendialog/issues/70

---

This has now been updated and the functionality moved to opendialogai/core#229.